build/tasks/release.js

```diff
@@ -49,6 +49,7 @@ gulp.task('bundle', function () {
   var bundles = [
     'app/cart/cart.component',
     'app/checkout/checkout.component',
+    'app/thank-you/thank-you.component',
```

Comment:
I went with camelCase on everything since it looked like that was what the project was using initially.